api/types/filters/parse_test.go

```diff
@@ -90,15 +90,15 @@ func TestFromParam(t *testing.T) {
 		`{"key": "value"}`,
 	}
 	valid := map[*Args][]string{
-		&Args{fields: map[string]map[string]bool{"key": {"value": true}}}: {
+		{fields: map[string]map[string]bool{"key": {"value": true}}}: {
```

Comment:
How did this run before our work? I don't think we've touched these files

Reply:
True.